src/sickle.ts

```diff
@@ -114,17 +178,20 @@ class Annotator {
       this.visit(child);
       lastEnd = child.getEnd();
     }
+    // Write any trailing text.
```

Comment:
lol, Daria had exactly the same bug, in the basically the same code. I should have caught it in the first PR pass.